### DISCUSSION TITLE
My github account has been suspended without giving any reason. What should I do?

### DISCUSSION THREAD

**Discussion by ramgoel1** [community/community#84107] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

on 2/1/2024, My github account has been suspended and I am reaching out to support via email for 2 days but there is no response. The github account has everything which I have achieved in tech till now.

Deleted account username - RamGoel
Ticket ID - 2508635

I can't lose the account. I request anyone to please suggest what should I do in this case?

**Comment by darkotodoric** [comment]
I'm sorry to hear that. I hope it's just a mistake. If they haven't responded yet, you can try reaching out to them on [Twitter](https://twitter.com/github). Some users have had success getting in touch with support that way.

**Reply by ramgoel1** [reply]
Hi @darkotodoric , thanks for the suggestion. My account has been enabled again.

**Reply by SAHELISAHAA** [reply]
@ramgoel1 what process did you follow?

**Comment by AbhijeetMITWPU** [comment]
I'm currently writing this message to report that my account was suspended on 11/04/2024 without prior notice. I've sent an email regarding this issue, including my ticket ID (2724927) and 
account username **(Abhijeetsingh610)** for reference. I

t's crucial for me to resolve this matter promptly as losing access to my account is concerning.

**Comment by NeuralFalconYT** [comment]
My account was suspended 7 days ago. My username is 'neuralfalcon'. I contacted to the support team and received a ticket, but I haven't gotten a response.  Can anyone help me?

**Reply by SAHELISAHAA** [reply]
@NeuralFalconYT current status?

**Comment by NeuralFalconYT** [comment]
I lost my GitHub account.
